scripts/pylib/pytest-twister-harness/src/twister_harness/device/hardware_adapter.py

```diff
@@ -93,6 +93,11 @@ def _prepare_runner_args(self) -> tuple[list[str], list[str]]:
 
     def _flash_and_run(self) -> None:
         """Flash application on a device."""
+        if self.device_config.skip_flash:
```

Comment:
Consider moving this skip to `DeviceAdapter:launch()` method:

device_adapter.py, line 80
```
        if self.device_config.skip_flash:
            logger.info('Skip flashing')
            self.connect()
        elif self.device_config.flash_before:
         ...
```

Reply:
done